config/initializers/logging.rb

```diff
@@ -0,0 +1,20 @@
+config = Rails.application.config
+
+if ENV["RAILS_LOG_TO_STDOUT"].present?
+  # heroku and docker: dump everything to stdout
+  config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
+elsif ENV["RAILS_LOG_TO_SYSLOG"]
```

Comment:
I want to be consistent in how we handle boolean environment variables.  In our .env.example we have some values that are 1 or 0, some that have values like "true", and apparently some that are enabled if the environment variable is set at all.  That's confusing.

This is outside the scope of this PR, but I'm bringing it up so that we can try to get some consistency.